src/internal/query.ts

```diff
@@ -160,12 +164,12 @@ export class Query<T = any> {
    * @param {IAggField} agg an aggregation object
    * @returns {string} compiled aggregation function
    */
-  private compileAggregation(agg: IAggField): string {
-    if (agg.fn === "COUNT" && agg.col === "*") {
+  private compileAggregation(agg: IAggField): string | ICompiledAggField {
```

Comment:
The return type here would be always `ICompiledAggField` right?

Reply:
right 👍